scripts/download-realm.js

```diff
@@ -23,9 +23,11 @@ const path = require('path');
 const os = require('os');
 const child_process = require('child_process');
 const fetch = require('node-fetch');
-const ini = require('ini').parse;
+const ini = require('ini');
 const decompress = require('decompress');
 
+const MANIFEST_FILENAME = '.manifest.list';
```

Comment:
maybe rename this file to .download-realm.list to better state what's its purpose and who is creating it

Reply:
I had the same thought after submitting. Actually I wondered about making in `download-realm.lock` like Cocoapods / yarn / npm, and maybe in JSON format?

Reply:
yep `download-realm.lock` seems fine. format can be either json or ini, whichever is prefered. I think we should gitignore it as well.